### PR TITLE
Update global anthropogenic CH4 emissions to EDGARv7.0

### PR DESCRIPTION
**Pull request by msulprizio**
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard/GCST

### Confirm you have reviewed the following documentation

- [x] [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html)

### Describe the update

The global anthropogenic emissions inventory for CH4 has been updated to EDGARv7.0 from EDGARv6.0. EDGARv7 emissions are provided as annual sector-specific files, while EDGARv6 emissions are monthly. To apply seasonality to EDGARv7, we computed monthly scale factors from the EDGARv6 emissions and apply apply them to EDGARv7 via HEMCO.

Also in this update:

- Fixed logicals for reading manure and rice scale factors. Only the GEPA, Scarpelli_Mexico, and Scarpelli_Canada  inventories use them. EDGARv6 does not.
- Fixing typo on the NEI99_DOW_CO scale factor path in HEMCO_Config.rc.carbon for GCClassic.
- Renaming EDGARv6 entries in HEMCO_Config.rc from `CH4_*` to `EDGAR6_CH4_*` to differentiate from EDGARv7.
- Fixing hierarchies for regional inventories in HEMCO_Config.rc.carbon for GCHP so they are consistent with GCClassic.

### Expected changes

This update will only impact CH4 and carbon simulations by updating global anthropogenic emissions to the more recent EDGARv7 that extends to 2021 (where EDGARv6 only went to 2018). Regional inventories for the US, Canada, and Mexico still overwrite some sectors.

### Reference(s)

See the EDGARv7.0 website: https://edgar.jrc.ec.europa.eu/dataset_ghg70.

### CODE REVIEW

File: run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon [hunk 1]
```diff
@@ -1232,10 +1280,37 @@ ${RUNDIR_CO2_COPROD}
...
 # --- Seasonal scaling factors ----
 #==============================================================================
-(((EDGARv6.or.GEPA
+(((GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
 10 MANURE_SF $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Manure_ScalingFactors.WithClimatology.nc  sf_ch4 2008-2016/1-12/1/0 C xy 1 1
 11 RICE_SF   $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Rice_ScalingFactors.SetMissing.nc         sf_ch4 2012/1-12/1/0  C xy 1 1
-)))EDGARv6.or.GEPA
+)))GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
+
+(((EDGARv7
```
Comment (yantosca) [inline run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon:1288]: Same question as for HEMCO_Config.rc.CH4 above (does EDGARv6 have a scale factor block)

File: run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon [hunk 2]
```diff
@@ -1303,10 +1358,37 @@ ${RUNDIR_CO2_COPROD}
 #==============================================================================
 # --- Seasonal scaling factors ----
 #==============================================================================
-(((EDGARv6.or.GEPA
+(((GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
 10 MANURE_SF $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Manure_ScalingFactors.WithClimatology.nc  sf_ch4 2008-2016/1-12/1/0 C xy 1 1
 11 RICE_SF   $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Rice_ScalingFactors.SetMissing.nc         sf_ch4 2012/1-12/1/0  C xy 1 1
-)))EDGARv6.or.GEPA
+)))GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
+
```
Comment (yantosca) [inline run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon:1365]: Same question as mentioned above.

File: run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon (hunk 1 above)
Comment (msulprizio) [reply run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon:1288]: See comment above.

File: run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon (hunk 2 above)
Comment (msulprizio) [reply run/GCHP/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon:1365]: See comment above.

**Pull request merged**